src/graphql.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { createGraphQLMiddleware } from './graphql';
+import { createSchema } from './schema';
+import { mocked } from 'jest-mock';
+import { GraphQLSchema } from 'graphql';
+
+jest.mock('./schema');
+
+describe('graphql', () => {
+  beforeEach(() => {
+    mocked(createSchema).mockResolvedValue(new GraphQLSchema({}));
+  });
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  describe('createGraphQLMiddleware', () => {
+    describe('with no options', () => {
+      it('returns a middleware with generated schema', async () => {
+        const middleware = createGraphQLMiddleware();
+        expect(middleware).resolves.toEqual(expect.any(Function));
```

Comment:
[Missing await](https://jestjs.io/docs/asynchronous#asyncawait)

```suggestion
        await expect(createGraphQLMiddleware()).resolves.toEqual(expect.any(Function));
```